doc/conf.py

```diff
@@ -527,6 +527,7 @@
     'theme_logo_only': False,
     'current_version': version,
     'versions': (("latest", "/"),
+                 ("2.2.0", "/2.2.0/"),
```

Comment:
can you also remove one from the bottom here, so we got to display the same number of releases?

Reply:
sure

Reply:
actually, I could remove two versions since 2.1 didn't remove any

Reply:
or perhaps even three, so we start the list with the LTS release?

Reply:
that's also fine with me.